pipeline-templates/terraform-plan.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         displayName: Plan - ${{ parameters.zone }}
         inputs:
           command: 'plan'
-          commandOptions: '-out=${{ parameters.zone }}${{ parameters.build }}plan'
+          commandOptions: '-out=${{ parameters.zone }}${{ parameters.build }}plan  -parallelism=50'
```

Comment:
@timja this saves me an additional 2minutes

Reply:
Could be worth adding to private dns too?  Not sure what the build times are like thetr

Reply:
I'm also thinking platform terraform, specifically front door changed if we add this to the apply

Reply:
don't think it will help, there's not many resources there